dvc/repo/run.py

```diff
@@ -5,9 +5,9 @@
 @locked
 @scm_context
 def run(self, no_exec=False, **kwargs):
-    from dvc.stage import Stage
+    from dvc.dvcfile import Dvcfile
 
-    stage = Stage.create(self, **kwargs)
+    stage = Dvcfile.create_stage(self, **kwargs)
```

Comment:
Again, this feels weird (esp because of `Dvcfile(self, stage.path).dump(stage)` down below). The multistage PR still looks strange https://github.com/iterative/dvc/pull/3584/files#r407061889 . We could definitely do better.